mautic/mautic
# Remove html switch from textarea custom field.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        |  🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #13655 



## Description

Remove the "Allow HTML" property in the custom field of the `textarea`. Additionally, migrate the data that has the "Allow HTML" property set to true.

~~I don't have the permission to set the milestone but this will be part of 6.x.~~ There is no BC break. The existing data will be automatically migrated. 






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a custom field with type: Text: Paragraph.
3. The custom field shouldn't have the Allow HTML properties.
4. Also, this includes migration, which will essentially rename any custom fields that have the "Allow HTML" setting set to true in the databases.

Note: In 5.x, the Allow HTML is broken. Any newly created custom field with textarea will not set the Allow HTML properties in the database.
